requirements.txt

```diff
@@ -1,60 +1,60 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.7
```

Comment:
It would be best if you used Python 3.8 as that's the currently supported version

Reply:
Done